lastadapter/build.gradle

```diff
@@ -10,7 +11,18 @@ android {
     dataBinding.enabled true
 }
 
+afterEvaluate {
```

Comment:
This is a workaround as suggested by Yigit Boyar [here](https://issuetracker.google.com/issues/70915745#comment8), and actual change [here](https://issuetracker.google.com/issues/70915745#comment11)

Reply:
this is not needed anymore, at least in 3.1 beta 1